doc/contributing/hw/racl/README.md

```diff
@@ -1,64 +1,92 @@
+Each `WritePerm` bit indicates if the corresponding role has write permission.
+When the bit is 1, the corresponding permission is granted and when it is 0 it is denied.
 
+The policy registers shall be mapped as shown below.
+Note, the policy registers shall be 64-bit aligned to allow room for future use.
+
+| Name     | Offset | Bits 31:16  | Bits 15:0  |
+|----------|:------:|:-----------:|:----------:|
+| Policy 0 | 0x000  | `writePerm` | `readPerm` |
+|          | 0x004  | reserved    | reserved   |
+| Policy 1 | 0x008  | `writePerm` | `readPerm` |
+|          | 0x00c  | reserved    | reserved   |
+| Policy 2 | 0x010  | `writePerm` | `readPerm` |
+|          | 0x014  | reserved    | reserved   |
+| Policy 3 | 0x018  | `writePerm` | `readPerm` |
+|          | 0x01c  | reserved    | reserved   |
+| Policy 4 | 0x020  | `writePerm` | `readPerm` |
+|          | 0x024  | reserved    | reserved   |
+
+In the [RACL configuration file](#top-level-racl-configuration), exactly one policy must be defined with `rot_private: true`.
+This policy is then used to statically protect the policy registers themselves.
+This means that the read and write permissions are used from this policy as defined in the configuration file at design time.
+For all policy registers, the configuration specified in the configuration file is also the reset default value.
+However, depending on the permissions defined for the `rot_private`-designated policy, the policy registers may also be updated at runtime.
+
+### RACL Controller
+
+The comportable IP [`racl_ctrl` (RACL Controller)](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/README.md) is the interface between the TileLink bus and the RACL matching logic.
+The RACL Controller [implements the policy registers](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/doc/registers.md), distributes them to subscribing IPs and collects error logs from them through its [hardware interface](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/doc/interfaces.md)
+
+### RACL Controller Groups
+A RACL controller group is defined by a name and a set of policies.
+Each group must also have one instance of a RACL controller associated with this group.
+Currently, only a single group is supported, but this is expected to change in the future.
+Any RACL-enabled IP subscribing to a specific RACL controller group is automatically connected to the associated RACL controller to get the appropriate policies and to collect the error logs from that specific group.
+
+## RACL Rules
+1. Each instantiation of a RACL-enabled IP, must state which [RACL Controller Group](#racl-controller-groups) it subscribes to.
+2. Each instantiation of a RACL-enabled IP, must state whether RACL is for a given bus interface.
+3. For each bus interface where RACL is enabled, each of its registers and windows must be mapped to a RACL policy that exists within the RACL Controller's Group which is used for this IP.
```

Comment:
Nit: I think you want "Controller Group" here (so it's not just some group that happens to belong to `racl_ctrl` :-D)